cutil/logger/logger.go

```diff
@@ -137,6 +125,20 @@ func Warning(format string, a ...interface{}) {
 	}
 }
 
+func extractLoggerArgs(format string, a ...interface{}) ([]interface{}, io.Writer) {
```

Comment:
I'll open up an issue for this. Let's use it to collect some ideas. I don't really feel the need for having the logger moved into its own repository really. While the logger is a central piece, as @prateekgogia rightfully pointed out, we don't necessarily need to invent our own solution. I'll share something in the issue right away. According to the rules, seems like we need 1 more LGTM here :-)